js/ui/map.js

```diff
@@ -287,6 +287,50 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Set the map's minimum zoom level, and zooms map to that level if it is currently below it. If no parameter provided, unsets the current minimum zoom (sets it to 0)
+     * @param {zoom} any number between 0 and 20
+     * @returns {Map} `this`
+     */
+    setMinZoom: function(minZoom) {
+
+        var zoom = minZoom === undefined ? 0 : minZoom;
```

Comment:
Apologies for being a little late to this discussion.

 In the similar `setMapBounds` function, we decided on accepting either `null` or `undefined` values for this argument ([see the conversation](https://github.com/mapbox/mapbox-gl-js/pull/2234#discussion_r55750632)). 

To unset the `minZoom`, we want users to write
```js
map.setMinZoom(null) // clear and self-documenting
```

We do NOT want users to write
```js
map.setMinZoom() // wat?
```